src/NWCClient.ts

```diff
@@ -168,6 +175,21 @@ export type Nip47SignMessageRequest = {
   message: string;
 };
 
+export type Nip47CreateConnectionRequest = {
```

Comment:
Needs to be updated to match hub changes: https://github.com/getAlby/hub/pull/1016/commits/4486aac309478789d20b6d00be77d849ce1f4e96